build.gradle

```diff
@@ -360,6 +362,8 @@ project(':ambry-frontend') {
                 project(':ambry-rest'),
                 project(':ambry-router')
         compile "io.dropwizard.metrics:metrics-core:$metricsVersion"
+        runtimeOnly "log4j:log4j:$log4jVersion"
```

Comment:
The open source main method for the frontend is actually in the ambry-rest package. It is called `RestServerMain`.

Reply:
+1